estuary/utils/story.py

```diff
@@ -0,0 +1,472 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+import abc
+
+from neomodel import db
+
+from estuary.error import ValidationError
+from estuary.models.koji import ContainerKojiBuild, KojiBuild, ModuleKojiBuild
+from estuary.models.bugzilla import BugzillaBug
+from estuary.models.distgit import DistGitCommit
+from estuary.models.errata import Advisory, ContainerAdvisory
+from estuary.models.freshmaker import FreshmakerEvent
+
+
+class BaseStoryManager(object):
+    """A class containing utility methods to create a story for an artifact."""
+
+    @staticmethod
+    def get_story_manager(item, config, limit=False):
+        """
+        Select which story flow to follow.
+
+        :param node item: a Neo4j node whose story is requested by the user
+        :param flask.config.Config config: flask config
+        :kwarg bool limit: specifies if LIMIT keyword should be added to the created cypher query
+        :return: instance of one of the story manager classes
+        :rtype: ModuleStoryManager/ContainerStoryManager
+        """
+        for class_name in config.get('STORY_MANAGER_SEQUENCE', ['ContainerStoryManager']):
+            story_manager_cls = globals().get(class_name)
```

Comment:
I'm sorry for being inconsistent here but after thinking about it, I think having a static variable that is a list of story manager classes would be better than just using `globals()`. Then in the error message below, you can say what the available classes are. This would also help in PR #192 with validating input from the user.